core/platform/PlatformMacros.h

```diff
@@ -444,15 +465,25 @@ public:                                                             \
  */
 #if __has_builtin(__builtin_expect)
 #    ifdef __cplusplus
-#        define UTILS_LIKELY(exp) (__builtin_expect(!!(exp), true))
+#        define UTILS_LIKELY(exp)   (__builtin_expect(!!(exp), true))
 #        define UTILS_UNLIKELY(exp) (__builtin_expect(!!(exp), false))
 #    else
-#        define UTILS_LIKELY(exp) (__builtin_expect(!!(exp), 1))
+#        define UTILS_LIKELY(exp)   (__builtin_expect(!!(exp), 1))
 #        define UTILS_UNLIKELY(exp) (__builtin_expect(!!(exp), 0))
 #    endif
 #else
-#    define UTILS_LIKELY(exp) (!!(exp))
+#    define UTILS_LIKELY(exp)   (!!(exp))
 #    define UTILS_UNLIKELY(exp) (!!(exp))
 #endif
 
+#if defined(_MSC_VER)
+// MSVC does not support loop unrolling hints
+#    define UTILS_UNROLL
+#    define UTILS_NOUNROLL
```

Comment:
Shouldn't these be with `AX_` prefix? And probably could be shortened to just `AX_UNROLL` and `AX_NOUNROLL`.

Reply:
> Shouldn't these be with `AX_` prefix? And probably could be shortened to just `AX_UNROLL` and `AX_NOUNROLL`.

yes, should be better